pkg/remediator/crashloopbackoffrescheduler_test.go

```diff
@@ -134,6 +134,21 @@ func (suite *TestCrashLoopBackOffReschedulerSuite) TestKeepsWithOtherReason() {
 	suite.run()
 }
 
+func (suite *TestCrashLoopBackOffReschedulerSuite) TestKeepsOptedOutUnHealthyPods() {
+	suite.pods[0].ObjectMeta.Annotations["kube-remediator/CrashLoopBackOffRemediator"] = "false"
+	suite.mockClient.EXPECT().GetPods("").Return(&corev1.PodList{Items: suite.pods}, nil).Times(1)
+
+	suite.run()
+}
+
+func (suite *TestCrashLoopBackOffReschedulerSuite) TestReschedulesUnHealthyPodsByDefault() {
+	delete(suite.pods[0].ObjectMeta.Annotations, "kube-remediator/CrashLoopBackOffRemediator")
```

Comment:
would prefer to modify the default case, remove the annotation and this test 🤷‍♂